src/env/index.ts

```diff
@@ -0,0 +1,19 @@
+import { TFabricEnv } from './types';
+import { getEnv as getBrowserEnv } from './browser';
+
+let env: TFabricEnv;
+
+export const setEnv = (value: TFabricEnv) => {
+  env = value;
+};
+
+export const getEnv = () => env || getBrowserEnv();
```

Comment:
i m lost a bit on this thing here.
Why again do we need an index file and the setEnv part if we could use the same trick with `isInitialized`

Seems like we want fabric importing from here, but having the import statement over the env/node file create a value with setEnv that nullify the browser env. Is that correct?